src/examples/drawer/DrawerDefault.tsx

```diff
@@ -20,7 +20,7 @@ const Example = () => {
     <Row>
       <Col textAlign="center">
         <Button onClick={open}>Open Drawer</Button>
-        <DrawerModal isOpen={isOpen} onClose={close}>
+        <DrawerModal isOpen={isOpen} onClose={close} restoreFocus focusOnMount>
```

Comment:
This isn't working for me. When I pull the example into CodeSandbox, only `restoreFocus` is needed to restore keyboard focus back to the trigger button (just like the `Modal` example).

Reply:
Thats correct, I am in the process of investigating as to why this isn't working as expected and it looks like focus is reverting to `#gatsby-focus-wrapper` for some odd reason. I will continue to look more into this.

Reply:
Did you try in production mode? I seem to remember we might've seen some odd Gatsby stuff with this in development.

Reply:
It didn't seem to work in development mode as well...